tests/test_atomicwrites.py

```diff
@@ -65,3 +66,24 @@ def test_open_reraise(tmpdir):
             f.name = "asdf"
             # Now trigger our own exception.
             assert False, "Intentional failure for testing purposes"
+
+
+def test_atomic_write_in_pwd(tmpdir):
+    orig_curdir= os.getcwd()
```

Comment:
Hmm. It seems that `tox -e py-stylecheck`, which I was using as recommended by CONTRIBUTING.rst, doesn't catch this (and possibly doesn't run anything at all), while `tox -e py35-stylecheck` does. Anyway, I'll fix it.